src/main/scala/monix/nio/tcp/AsyncSocketChannel.scala

```diff
@@ -23,103 +23,354 @@ import java.nio.channels.spi.AsynchronousChannelProvider
 import java.nio.channels.{ AsynchronousChannelGroup, AsynchronousSocketChannel, CompletionHandler }
 import java.util.concurrent.TimeUnit
 
-import monix.eval.Callback
+import monix.eval.{ Callback, Task }
 import monix.execution.Scheduler
 import monix.nio.internal.ExecutorServiceWrapper
 
+import scala.concurrent.duration.Duration
+import scala.concurrent.{ Future, Promise }
 import scala.util.control.NonFatal
 
+/**
```

Comment:
I had it enabled. scalariform defaults to placing the asterisks beneath the first. Changed the setting 

> case object PlaceScaladocAsterisksBeneathSecondAsterisk extends BooleanPreferenceDescriptor {
  val key = "placeScaladocAsterisksBeneathSecondAsterisk"
  val description = "Place Scaladoc asterisks beneath the second asterisk in the opening '/**', as opposed to the first"
  val defaultValue = **false**
}